dashboard/modules/reporter/reporter_agent.py

```diff
@@ -98,7 +103,16 @@ def jsonify_asdict(o):
     "raylet_cpu": Gauge("raylet_cpu", "CPU usage of the raylet on a node.",
                         "percentage", ["ip", "pid"]),
     "raylet_mem": Gauge("raylet_mem", "Memory usage of the raylet on a node",
-                        "mb", ["ip", "pid"])
+                        "mb", ["ip", "pid"]),
+    "cluster_active_nodes": Gauge("cluster_active_nodes",
+                                  "Active nodes on the cluster", "count",
+                                  ["ip", "node_type"]),
```

Comment:
actually we don't need ip here because we are only reporting in the head node

Reply:
removed ip from active nodes, we don't need the ip for the failed nodes or pending nodes record either right?